src/shared/ui/AppletRoundLogo.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react-native/no-inline-styles */
+import React, { FC } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { Text } from '@tamagui/core';
+import { CachedImage } from 'react-native-img-cache';
+import Svg, { Circle, Defs, LinearGradient, Stop } from 'react-native-svg';
+
+import { Applet } from '@entities/applet/lib';
+
+import { colors } from '../lib';
+
+import { Box } from '.';
+
+declare module 'react-native-svg' {
+  interface LinearGradientProps {
+    children: JSX.Element | JSX.Element[];
+  }
+}
+
+type Props = {
+  applet: Applet;
+  size?: number;
+};
```

Comment:
As far as I understand, the types that are actually in use (from the whole `Applet` type): "image", "name".
Let's abstract from "Applet" type here and extend Props like this: 
```
type Props = {
  imageURI: string;
  name: string;
  size?: number;
}
``